.travis/smoke-test.sh

```diff
@@ -3,9 +3,12 @@ set -ex
 
 COMPOSE_PROJECT_NAME="${COMPOSE_PROJECT_NAME:-sample-configuration}"
 HTTP_PORT="${HTTP_PORT:-8080}"
+HTTP_PORT_GATEWAY="${HTTP_PORT_GATEWAY:-8081}"
 
 echo "Wait for containers health"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_dummy-api_fpm_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_scholarly-articles_fpm_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_scholarly-articles_web_1"
```

Comment:
Seem to have an existing order of `_web` at the bottom.

Reply:
I was going more for application-by-application but it's really about dependencies so I'll try to re-order things as they make sense.